astropy/units/format/generic.py

```diff
@@ -469,11 +469,14 @@ def parse(cls, s, debug=False):
             s = s.decode('ascii')
 
         result = cls._do_parse(s, debug=debug)
-        if s.count('/') > 1:
-            warnings.warn(
-                "'{}' contains multiple slashes, which is "
-                "discouraged by the FITS standard".format(s),
-                core.UnitsWarning)
+        nsolidus = s.count('/')
+        if nsolidus > 1:
+            # Check for fractional exponents (accepted)
+            if nsolidus - len(re.findall(r'\(\d+/\d+\)', s)) > 1:
```

Comment:
Is nested `if` necessary? Can this be flattened to `if this and that`?

Reply:
Hadn't updated the page with your comment, but I was musing about this at the same time ;-)